lib/pacemaker/pcmk_sched_colocation.c

```diff
@@ -1474,9 +1473,11 @@ pcmk__add_colocated_node_scores(pe_resource_t *rsc, const char *log_id,
         flags |= pcmk__coloc_select_active;
 
         for (GList *iter = colocations; iter != NULL; iter = iter->next) {
-            pe_resource_t *other = NULL;
             pcmk__colocation_t *constraint = (pcmk__colocation_t *) iter->data;
 
+            pe_resource_t *other = NULL;
+            float other_factor = factor * constraint->score / (float) INFINITY;
```

Comment:
I can't think of any unwanted side effects for "this with" colocations (this commit is aimed at "with this"), and obviously regression passed; hopefully there aren't any